ai2thor/tests/test_unity.py

```diff
@@ -276,6 +274,23 @@ def test_teleport(controller):
     position = controller.last_event.metadata['agent']['position']
     assert_near(position, dict(x=-2.0, z=-2.5, y=0.901))
 
+@pytest.mark.parametrize("controller", [wsgi_controller, fifo_controller])
+def test_open(controller):
```

Comment:
This tests trying consecutive openness values in [0.5, 0.7, 0]. Here, passing in `openness=0` is exactly what is done with `CloseObject`.

Reply:
LGTM.